USAGE.md

```diff
@@ -231,6 +231,33 @@ Additionally, you can also convert output to pdf or other types supported by uno
 5031
 ```
 
+## Inline images
+
+The `docx-template` engine supports including inline images. Here is shown how one can
```

Comment:
The local filename may not be the same as the one used in the template. Maybe add some documentation on how to "simulate" the filename that's transmitted, or just don't use the shortest way possible (use the syntax from request's example here https://requests.readthedocs.io/en/master/user/quickstart/#post-a-multipart-encoded-file )

Reply:
Done!